drivers/display/ssd16xx.c

```diff
@@ -60,6 +60,14 @@ struct ssd16xx_quirks {
 	 * SSD16XX_CMD_UPDATE_CTRL2 for a partial refresh.
 	 */
 	uint8_t ctrl2_partial;
+
+	/*
+	 * Device specific flag deciding whether to pass in the byte address or the bit address
+	 * when setting the RAM x position window or counter in SSD16XX_CMD_RAM_XPOS_CTRL or
+	 * SSD16XX_CMD_RAM_XPOS_CNTR.
+	 * Expected to be `true` for the devices that expects the byte address, `false` otherwise.
+	 */
```

Comment:
Please follow the style in `“nearby” code`.
```
        /*
         * Device specific flag deciding whether to pass in the byte address or
         * the bit address when setting the RAM x position window or counter in
         * SSD16XX_CMD_RAM_XPOS_CTRL or SSD16XX_CMD_RAM_XPOS_CNTR. Expected to
         * be `true` for the devices that expects the byte address, `false`
         * otherwise.
         */
```